packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/bin/server/controllers/push_notifications_controller.dart

```diff
@@ -80,9 +82,12 @@ class PushNotificationsController extends ApiController {
       message,
       BadRequestException('Push message can not be empty.'),
     );
+    if (!(_pushTokens.tokens.any((element) => element.token == pushToken))) {
+      throw BadRequestException('Notifications disabled by the user');
```

Comment:
This is pretty minor - usually `400 Bad Request` is used to indicate a client error like malformed request syntax. I think something like `404 Not Found` or `410 Gone` will be more suitable, since the push token is missing.